src/Models/User/User.php

```diff
@@ -126,7 +126,7 @@ public function groups(): BelongsToMany
     public function isFreeloader(): bool
```

Comment:
We should also have an explicit function that lists all freeloaded ShiftEntries that a user freeloaded, something like

```
    public function siftEntriesFreeloadedBy(): HasMany
    {
        return $this->hasMany(ShiftEntry::class); // needs additional parameters to check the right target field
    }

Reply:
out of scope for this PR
this PR is just refactoring and to show freeloaded by in user my shifts view